Tests/StreamChatTests/Query/Sorting/ListDatabaseObserver+Sorting.swift

```diff
@@ -371,6 +384,20 @@ final class ListDatabaseObserver_Sorting: XCTestCase {
             ($0, $1, $1)
         })
     }
+
+    private func startObservingAndWaitForInitialUpdate(isBackground: Bool, file: StaticString = #file, line: UInt = #line) throws {
```

Comment:
This one has the same logic as `startObservingWaitingForInitialResults`, maybe we can reuse it?